modules/nf-core/crumble/main.nf

```diff
@@ -22,13 +21,13 @@ process CRUMBLE {
     task.ext.when == null || task.ext.when
 
     script:
-    def args      = task.ext.args ?: ''
-    def prefix    = task.ext.prefix ?: "${meta.id}"
-    def extension = args.contains("-O sam") ? "sam" :
+    def args       = task.ext.args ?: ''
+    def prefix     = task.ext.prefix ?: "${meta.id}"
+    def extension  = args.contains("-O sam") ? "sam" :
                     args.contains("-O bam") ? "bam" :
                     args.contains("-O cram") ? "cram" :
                     "sam"
-    def bed_output = bed ? "-b ${prefix}.bed" : ""
+    def bed_output = args.contains("-b") ? "-b ${prefix}.bed" : ""
```

Comment:
Just drop the `bed_output`, it's better if you use the full argument in `args` otherwise it might get a bit confusing for the user (you can access `meta` inside of `args`)

Reply:
You can access `meta` in `args` similar to [this](https://github.com/nf-core/modules/blob/6a09f1733c3eca3cfb7d6205b0c1ba49242a5a35/tests/modules/nf-core/gatk4/printsvevidence/nextflow.config#L6) (You usually define the prefix from `meta`, so this is fine)
I would suggest making `args` like this in the config file:
```
withName: CRUMBLE {
    ext.args = {"-O cram -b ${meta.id}.bed"}
}
```

(also is there a long notation of the arguments available?)

Reply:
Updated `meta.yml` with instructions as suggested. 

No long notation available unfortunately.